models/comments-model.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection.js");
+
+// TASK 9
+exports.fetchCommenstById = (id) => {
+  const valideIds = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13];
```

Comment:
what happens when someone posts a new comment? we can't get it because it would be id 14. You'll need to handle this differently.